api/app/controllers/application.go

```diff
@@ -47,13 +67,38 @@ type Application struct {
 func (a ApplicationController) AppPOST() gin.HandlerFunc {
 	return func(c *gin.Context) {
 		app := Application{
-			Name:  c.PostForm("name"),
-			Image: c.PostForm("image"),
+			Name:          c.PostForm("name"),
+			Image:         c.PostForm("image"),
+			ResourceGroup: c.PostForm("resourceGroup"),
```

Comment:
Question:

why do we have `resourceGroup` here? , not saying it shouldn't be here just don't know what it does

Reply:
Good question! The idea is that every resource has to belong to a resource group. so when you deploy something, you have to have a resource group. RIGHT NOW, you are not given a way to create a second resource group, and if you don't pass one in the `default` one is selected for you. If you give it one that does not exist, it will fail.

Reply:
Makes sense. Thanks for clearing that up!